src/components/RunHeader/RunHeader.scss

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 
 .pipeline-run-header {
   background: white;
+  margin: -3em -6% 2em -6%;
```

Comment:
This is a temporary hack to preserve the existing design while keeping the desired structure for a common header / nav since the RunHeader is no longer a child of the global Header.